draft-bbs-signatures.md

```diff
@@ -52,11 +52,13 @@ A digital signature scheme is a fundamental cryptographic primitive that is used
 
 However, traditional digital signature schemes require both the signature and the entire message to be disclosed during verification, constraining its usage in certain applications.
 
-The BBS Signature scheme provides many new properties however two ones are
+The BBS Signature scheme provides many new properties however three key ones are:
 
 **Selective Disclosure** - The scheme allows a signer to sign multiple payloads while still produce a single output signature. An intermediary then possessing the payloads and the signature can generate a proof whereby they can choose which payloads to reveal.
 
-**Unlinkable Presentations** - The proofs generated by the scheme are known as a signature proof of knowledge meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof.
+**Unlinkable Proof Presentations** - The proofs generated by the scheme are known as a signature proof of knowledge meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof.
+
+**Scoped Proofs** - The proofs generated by the scheme feature a presentation message where the proving party is able to include any information they would like to cryptographically tie to the proof. This can include a cryptographic nonce and or an audience/domain identifier to ensure the generated proof can only be used appropriately and provides a way for a verifying/relying party to detect a replay attack.
```

Comment:
Maybe 'prevent' a replay attack? Assuming the verifier chooses a sufficiently unique presentation message, a replayed proof would simply fail to verify, but the verifier wouldn't know the reason for the mismatch in the hash.

Reply:
I like detect because its what you choose to do with that information that leads to prevention. The scheme only provides you a way to be able to detect and therefore prevent.